lib/logstash/inputs/kafka.rb

```diff
@@ -146,7 +146,7 @@ class LogStash::Inputs::Kafka < LogStash::Inputs::Base
   # Time kafka consumer will wait to receive new messages from topics
   config :poll_timeout_ms, :validate => :number, :default => 100
   # Enable SSL/TLS secured communication to Kafka broker.
-  config :ssl, :validate => :boolean, :default => false
+  config :ssl, :validate => :boolean, :default => false, :deprecated => "Use security_protocol setting instead"
```

Comment:
Maybe have the deprecation more directly what the solution is:

"Use security_protocol => 'ssl'"